app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/network/RealVpnDetector.kt

```diff
@@ -29,12 +30,13 @@ import javax.inject.Inject
 @ContributesBinding(AppScope::class)
 class RealExternalVpnDetector @Inject constructor(
     private val context: Context,
-    private val vpnFeaturesRegistry: VpnFeaturesRegistry,
+    private val appTrackingProtection: AppTrackingProtection,
+    private val networkProtectionState: NetworkProtectionState,
 ) : ExternalVpnDetector {
 
     override suspend fun isExternalVpnDetected(): Boolean {
         // if we're the ones using the VPN, no VPN is detected
-        if (vpnFeaturesRegistry.isAnyFeatureRunning()) return false
+        if (appTrackingProtection.isRunning() || networkProtectionState.isRunning()) return false
```

Comment:
I am not sure I like this change. This will call through `vpnServiceWrapper.isServiceRunning()` two times and that call is very expensive. maybe we want to leave it unchaged